src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -7,6 +7,8 @@ Release History
 ++++++
 * Completions kick in as soon as command table loading is done.
 * Fix bug with using `--style` parameter.
+* Interactive lexer instantiated after command table dump if missing
+* Tweaks to completer scenarios
```

Comment:
Can you be more descriptive? I suspect this will be called out when the release notes are reviewed.